app/models/like.rb

```diff
@@ -1,2 +1,10 @@
 class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  private
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Please do the same here

Reply:
Noted.